test/unit/infura-controller-test.js

```diff
@@ -0,0 +1,34 @@
+// polyfill fetch
+global.fetch = global.fetch || require('isomorphic-fetch')
```

Comment:
I copied this from the currency controller test--if there's a better way to do this, let me know and we'll change it and I'll learn the new way!